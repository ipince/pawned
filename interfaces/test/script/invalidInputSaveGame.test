# Invalid input in SaveGame command
StartNewGame human 0 human 0
SaveGame
SaveGame this game
SaveGame this saved game