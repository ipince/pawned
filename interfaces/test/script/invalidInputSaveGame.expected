# Invalid input in SaveGame command
New game started
Input error
Input error
Input error
